data/hai/hai reveal 4 middle.txt

```diff
@@ -688,10 +688,13 @@ mission "Hai Reveal [A04] Report to Teeneep"
 	landing
 	name "Report to Teeneep"
 	description `Tell Teeneep about the experience with Scar's Legion.`
-	source Hai-home
+	source
+		government "Hai"
+		not planet Darkwaste
+		not attributes "uninhabited"
```

Comment:
This isn't necessary. While it feels like logically it should be able to fire anywhere, the previous thing ends on Hai-home so this should just trigger immediately after and save on unnecessary lines.

Reply:
I think you're right, but I'll retest without this to make sure.

Reply:
This needs "not planet Darkwaste" since you have to fly to Darkwaste. It does not need the other conditions.